### PR TITLE
Automating TACO file assembly

### PR DESCRIPTION
**Pull request by alexey-temnikov**
### Summary
Automating TACO file assembly

### Description
Refactored Docker scripts and added GitHub Action

### Related Issue
https://bitquill.atlassian.net/browse/AD-460

### Additional Reviewers
@affonsoBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

File: .github/workflows/tableau-connector.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Asssemble Tableau Connector
+on:
+  push:
+    branches: [ master, develop ]
```
Comment (affonsov) [inline .github/workflows/tableau-connector.yml]: use main instead of master

File: tableau-connector/build.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+fi
+CURRENT_FOLDER="${ABSOLUTE_PATH%/*}"
+echo "ABSOLUTEPATH=${ABSOLUTE_PATH}"
+echo "CURRENT_FOLDER=${CURRENT_FOLDER}"
+TARGET_FOLDER="$CURRENT_FOLDER"/target
+echo TARGET_FOLDER=${TARGET_FOLDER}
+mkdir -p $TARGET_FOLDER
+
+echo "Building Docker Image"
+docker build -t taco-builder $CURRENT_FOLDER
```
Comment (affonsoBQ) [inline tableau-connector/build.sh:17]: building two times make you lost the label of the previous image. need or to check if the image exists or a mechanism of versioning. create a version and then create a tag latest, or if there is a image delete before create a new one, or if there is a image to not delete

File: .github/workflows/tableau-connector.yml (hunk 1 above)
Comment (alexey-temnikov) [reply .github/workflows/tableau-connector.yml]: Fixed

**Review by affonsov — APPROVED** [review]
lgtm

**Pull request merged**